addons/mail/static/src/components/notification_popover/notification_popover.js

```diff
@@ -1,23 +0,0 @@
-/** @odoo-module **/
-
-import { registerMessagingComponent } from '@mail/utils/messaging_component';
-
-const { Component } = owl;
-
-export class NotificationPopover extends Component {
-
-    /**
-     * @returns {MessageView}
-     */
-    get messageView() {
-        return this.messaging && this.messaging.models['MessageView'].get(this.props.messageViewLocalId);
-    }
-
-}
-
-Object.assign(NotificationPopover, {
-    props: { messageViewLocalId: String },
-    template: 'mail.NotificationPopover',
```

Comment:
Your should also delete the xml template (and the folder since it will be empty)

Reply:
Done